### DISCUSSION TITLE
Behaviour of vertex_alignments and other arguments in periodic systems as the number of cells changes

### DISCUSSION THREAD

**Discussion by andrewtarzia** [lukasturcani/stk#429] [General]
Putting this discussion here to help us decide on behaviour, which I can then implement.

Currently, when you increase the lattice values of a Cof class, you get new distinct vertices (as you should) and, therefore, must add them to your dictionary of, for example, vertex_alignments.

Considering these vertices should be periodic images, do we think that is the right behaviour? @lukasturcani 

I can see benefits to either approach, hence it is a discussion, not an issue.

A minimum example to come!

**Comment by lukasturcani** [comment]
I think so, because this way you do not have to make *just* periodic images. So it is more flexible. I guess the downside is the repetition, though in general I do not mind that, that's what functions are for after all. 

Having said that, if there's a way to simplify the API / definition then I'm open to it. But it has to be balanced against the overhead of users understanding the additional complexity of such an API.

**Comment by andrewtarzia** [comment]
I agree. The more I think about this, the flexibility is key!
